src/utils/details.js

```diff
@@ -27,6 +28,15 @@ function _createRequest ({
   ])
 }
 
+function validateHeaders (headers) {
+  for (const name in headers) {
+    const value = headers[name].toString()
+    if (value && value.indexOf('\n') !== -1) {
```

Comment:
I think that would be too strict. This would for example not allow us to put URLs into a header.